packages/reactstrap-validation-select/src/AvSelect.js

```diff
@@ -259,6 +261,16 @@ class AvSelect extends AvBaseInput {
       Tag = Creatable;
     }
 
+     // Enhance placeholder for accessibility
+  placeholder = (
+    <>
+      {placeholder || 'Select...'}
+      <span className="sr-only">
+        {ariaMessage.ariaErrorDesc || null} {helpMessage || null}
```

Comment:
- we don't want an error message to appear until the field has been touched (the feedback doesn't appear until after you leave the field for the first time, so it's the same), so check if touched and hasError . 
- Since ariaMessage isn't required I'd make it `ariaMessage?.ariaErrorDesc`